<ui version="4.0" >
 <class>OptEvents</class>
 <widget class="QWidget" name="OptEvents" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptEventsUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QCheckBox" name="ck_popupMsgs" >
     <property name="text" >
      <string>Automatically open new messages</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_stealFocus" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Take focus when messages are opened</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_allowNonRoster" >
     <property name="text" >
      <string>Allow events from unknown contacts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Authorization</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ck_autoAuth" >
        <property name="text" >
         <string>Allow others to see my status automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_notifyAuth" >
        <property name="text" >
         <string>Notify me when others allow me to see their status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_bounce" >
       <property name="text" >
        <string>Bounce dock:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>cb_bounce</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_bounce" >
       <item>
        <property name="text" >
         <string>Never</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Once</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Forever</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>174</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Popup notifications</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>10</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ck_popupOnMessage" >
        <property name="text" >
         <string>On incoming event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_popupOnOnline" >
        <property name="text" >
         <string>When a contact becomes available</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_popupOnOffline" >
        <property name="text" >
         <string>When a contact becomes unavailable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_popupOnStatus" >
        <property name="text" >
         <string>When a contact changes status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>ck_popupMsgs</tabstop>
  <tabstop>ck_stealFocus</tabstop>
  <tabstop>ck_allowNonRoster</tabstop>
  <tabstop>ck_autoAuth</tabstop>
  <tabstop>ck_notifyAuth</tabstop>
  <tabstop>cb_bounce</tabstop>
  <tabstop>ck_popupOnMessage</tabstop>
  <tabstop>ck_popupOnOnline</tabstop>
  <tabstop>ck_popupOnOffline</tabstop>
  <tabstop>ck_popupOnStatus</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ck_popupMsgs</sender>
   <signal>toggled(bool)</signal>
   <receiver>ck_stealFocus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>90</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
